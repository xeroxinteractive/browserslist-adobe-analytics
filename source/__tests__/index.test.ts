import * as moduleUnderTest from '../index';

test('hello world', () => {
  expect(moduleUnderTest.default()).toBe('Hello World!');
});
