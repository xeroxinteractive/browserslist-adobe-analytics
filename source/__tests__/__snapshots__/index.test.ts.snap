// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello world 1`] = `
Object {
  "and_chr": Object {
    "78": 0.08700757666608093,
  },
  "and_ff": Object {
    "68": 0,
  },
  "and_qq": Object {
    "1.2": 0,
  },
  "and_uc": Object {
    "12.12": 0,
  },
  "android": Object {
    "2.1": 0,
    "2.2": 0,
    "2.3": 0,
    "3": 0,
    "4": 0,
    "4.1": 0,
    "4.2-4.3": 0,
    "4.4": 0,
    "4.4.3-4.4.4": 0,
    "76": 0,
  },
  "baidu": Object {
    "7.12": 0,
  },
  "bb": Object {
    "10": 0,
    "7": 0,
  },
  "chrome": Object {
    "10": 0,
    "11": 0,
    "12": 0,
    "13": 0,
    "14": 0,
    "15": 0,
    "16": 0,
    "17": 0,
    "18": 0,
    "19": 0,
    "20": 0,
    "21": 0,
    "22": 0,
    "23": 0,
    "24": 0,
    "25": 0,
    "26": 0,
    "27": 0,
    "28": 0,
    "29": 0,
    "30": 0,
    "31": 0,
    "32": 0,
    "33": 0,
    "34": 0,
    "35": 0,
    "36": 0,
    "37": 0,
    "38": 0,
    "39": 0,
    "4": 0,
    "40": 0,
    "41": 0,
    "42": 0,
    "43": 0,
    "44": 0,
    "45": 0,
    "46": 0,
    "47": 0,
    "48": 0,
    "49": 0.0040739448150082505,
    "5": 0,
    "50": 0,
    "51": 0,
    "52": 0,
    "53": 0,
    "54": 0,
    "55": 0,
    "56": 0,
    "57": 0.12370685414416077,
    "58": 0,
    "59": 0,
    "6": 0,
    "60": 0,
    "61": 0,
    "62": 0,
    "63": 0.0012799910718787185,
    "64": 0,
    "65": 0,
    "66": 0,
    "67": 0,
    "68": 0.015932584852639497,
    "69": 0.0011693685165270588,
    "7": 0,
    "70": 0.0011909056511972933,
    "71": 0.0014327089359040188,
    "72": 0.002160566191691267,
    "73": 0.002404816878065065,
    "74": 0.002607461736098636,
    "75": 0.004199741260695757,
    "76": 0.013378965953216448,
    "77": 0.3081782863587215,
    "78": 0.03772473877658306,
    "79": 0,
    "8": 0,
    "80": 0,
    "81": 0.007112638724844994,
    "9": 0,
  },
  "edge": Object {
    "12": 0,
    "13": 0,
    "14": 0,
    "15": 0,
    "16": 0.002519844756417454,
    "17": 0.008608490623759473,
    "18": 0.03671298292741545,
    "76": 0,
  },
  "firefox": Object {
    "10": 0,
    "11": 0,
    "12": 0,
    "13": 0,
    "14": 0,
    "15": 0,
    "16": 0,
    "17": 0,
    "18": 0,
    "19": 0,
    "2": 0,
    "20": 0,
    "21": 0,
    "22": 0,
    "23": 0,
    "24": 0,
    "25": 0,
    "26": 0,
    "27": 0,
    "28": 0,
    "29": 0,
    "3": 0,
    "3.5": 0,
    "3.6": 0,
    "30": 0,
    "31": 0,
    "32": 0,
    "33": 0,
    "34": 0,
    "35": 0,
    "36": 0,
    "37": 0,
    "38": 0,
    "39": 0,
    "4": 0,
    "40": 0,
    "41": 0,
    "42": 0,
    "43": 0,
    "44": 0,
    "45": 0,
    "46": 0,
    "47": 0,
    "48": 0.0009765132651617757,
    "49": 0,
    "5": 0,
    "50": 0,
    "51": 0,
    "52": 0.002611867059099366,
    "53": 0,
    "54": 0,
    "55": 0,
    "56": 0,
    "57": 0,
    "58": 0,
    "59": 0,
    "6": 0,
    "60": 0.0035692905912579793,
    "61": 0,
    "62": 0,
    "63": 0,
    "64": 0,
    "65": 0,
    "66": 0,
    "67": 0.000977981706162019,
    "68": 0.004767048967123075,
    "69": 0.052860449646423883,
    "7": 0,
    "70": 0.014139618391342463,
    "71": 0,
    "72": 0,
    "8": 0,
    "9": 0,
  },
  "ie": Object {
    "10": 0,
    "11": 0.1137953668728521,
    "5.5": 0,
    "6": 0,
    "7": 0,
    "8": 0,
    "9": 0,
  },
  "ie_mob": Object {
    "10": 0,
    "11": 0,
  },
  "ios_saf": Object {
    "10.0-10.2": 0,
    "10.3": 0,
    "11.0-11.2": 0.0014434775032391362,
    "11.3-11.4": 0,
    "12.0-12.1": 0.0013886557058967206,
    "12.2-12.4": 0,
    "13.0-13.1": 0,
    "13.2": 0,
    "13.3": 0,
    "3.2": 0,
    "4.0-4.1": 0,
    "4.2-4.3": 0,
    "5.0-5.1": 0,
    "6.0-6.1": 0,
    "7.0-7.1": 0,
    "8": 0,
    "8.1-8.4": 0,
    "9.0-9.2": 0,
    "9.3": 0,
  },
  "kaios": Object {
    "2.5": 0,
  },
  "op_mini": Object {
    "all": 0,
  },
  "op_mob": Object {
    "10": 0,
    "11": 0,
    "11.1": 0,
    "11.5": 0,
    "12": 0,
    "12.1": 0,
    "46": 0,
  },
  "opera": Object {
    "10.0-10.1": 0,
    "10.5": 0,
    "10.6": 0,
    "11": 0,
    "11.1": 0,
    "11.5": 0,
    "11.6": 0,
    "12": 0,
    "12.1": 0,
    "15": 0,
    "16": 0,
    "17": 0,
    "18": 0,
    "19": 0,
    "20": 0,
    "21": 0,
    "22": 0,
    "23": 0,
    "24": 0,
    "25": 0,
    "26": 0,
    "27": 0,
    "28": 0,
    "29": 0,
    "30": 0,
    "31": 0,
    "32": 0,
    "33": 0,
    "34": 0,
    "35": 0,
    "36": 0,
    "37": 0,
    "38": 0,
    "39": 0,
    "40": 0,
    "41": 0,
    "42": 0,
    "43": 0,
    "44": 0,
    "45": 0,
    "46": 0,
    "47": 0,
    "48": 0,
    "49": 0,
    "50": 0,
    "51": 0,
    "52": 0,
    "53": 0,
    "54": 0,
    "55": 0,
    "56": 0,
    "57": 0,
    "58": 0,
    "60": 0,
    "62": 0,
    "63": 0.005230097362533119,
    "64": 0.0012139112268677713,
    "9": 0,
    "9.5-9.6": 0,
  },
  "safari": Object {
    "10": 0,
    "10.1": 0,
    "11": 0,
    "11.1": 0.0063652022557211686,
    "12": 0,
    "12.1": 0.017297256022198913,
    "13": 0.06995457133025582,
    "3.1": 0,
    "3.2": 0,
    "4": 0,
    "5": 0,
    "5.1": 0,
    "6": 0,
    "6.1": 0,
    "7": 0,
    "7.1": 0,
    "8": 0,
    "9": 0,
    "9.1": 0,
    "TP": 0,
  },
  "samsung": Object {
    "10.1": 0.005907048663645268,
    "4": 0,
    "5.0-5.4": 0,
    "6.2-6.4": 0,
    "7.2-7.4": 0,
    "8.2": 0,
    "9.2": 0,
  },
}
`;
